.taskcluster.yml

```diff
@@ -126,7 +115,7 @@ tasks:
                   routes:
                       $flatten:
                           - checks
-                          - $if: 'level == "3"'
+                          - $if: 'tasks_for != "github-pull-request"'
```

Comment:
This should use the `isPullRequest` variable in case the PR policy ever switches to `public_restricted` (in which cases tasks for will be "github-pull-request-untrusted")

Reply:
Yes, that's a good point - thank you.